pkg/aws/wait.go

```diff
@@ -159,3 +161,23 @@ func waitForClusterOperators(cfg *rest.Config) error {
 	_, err = clientwatch.UntilWithSync(ctx, listWatcher, &configapi.ClusterOperator{}, nil, clusterOperatorsAreAvailable)
 	return err
 }
+
+func waitForServiceLoadBalancerDNS(client kubeclient.Interface, namespace, name string) (string, error) {
+	var serviceDNS string
+
+	err := wait.PollImmediate(10*time.Second, serviceLoadBalancerTimeout, func() (bool, error) {
```

Comment:
It's preferable to use `clientwatch.UntilWithSync` as in https://github.com/openshift/hive/blob/5f17ab3a5fc96db6c670f81c105724213fcae4a1/contrib/cmd/waitforjob/main.go#L118
or 
https://github.com/openshift/hive/pull/953/files#diff-0a8cf6f710f69aa0899ff66b54ce3951R475